homeassistant/components/pioneer/media_player.py

```diff
@@ -31,14 +34,16 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
     vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): cv.socket_timeout,
+    vol.Optional(CONF_SOURCES, default=DEFAULT_SOURCES):
+        {cv.string: cv.string},
 })
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Pioneer platform."""
     pioneer = PioneerDevice(
         config.get(CONF_NAME), config.get(CONF_HOST), config.get(CONF_PORT),
-        config.get(CONF_TIMEOUT))
+        config.get(CONF_TIMEOUT), config.get(CONF_SOURCES))
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
I'm sorry but i have to ask: is the requested change to write `config[CONF_SOURCES]` instead? Should i do the same for all the other, already existing `config.get`s (for consistency)?

Reply:
Yes, please, if they follow the criteria above.

Reply:
As far as i understand, all config keys have either a default value or are mandatory, so i changed all the `config.get` calls to the suggested syntax on 4d4e4d554